diff --git a/CMakeLists.txt b/CMakeLists.txt
index 60096a8..4e5eeb6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,6 +11,7 @@ option(BUILD_SHARED_LIBS "Build shared library, disable for building the static
 option(LIBLCF_WITH_ICU "ICU encoding handling (when disabled only windows-1252 is supported, default: ON)" ON)
 option(LIBLCF_WITH_XML "XML reading support (expat, default: ON)" ON)
 option(LIBLCF_UPDATE_MIMEDB "Whether to run update-mime-database after install (default: ON)" ON)
+option(LIBLCF_FOR_SERVER "Compile only the source files needed for the server (default: OFF)" OFF)
 option(LIBLCF_ENABLE_TOOLS "Whether to build the tools (default: ON)" ${LIBLCF_MAIN_PROJECT})
 option(LIBLCF_ENABLE_TESTS "Whether to build the unit tests (default: ON)" ${LIBLCF_MAIN_PROJECT})
 option(LIBLCF_ENABLE_BENCHMARKS "Whether to build the benchmarks (default: OFF)" OFF)
@@ -292,7 +293,14 @@ set(LCF_HEADERS
 	src/lcf/third_party/string_view.h
 )
 
+if(LIBLCF_FOR_SERVER)
+set(LCF_SOURCES_SERVER
+	dummy.cpp
+)
+add_library(lcf ${LCF_SOURCES_SERVER} ${LCF_HEADERS})
+else()
 add_library(lcf ${LCF_SOURCES} ${LCF_HEADERS})
+endif()
 
 # IDE source grouping
 foreach(SG LDB LMT LMU LSD RPG THIRD_PARTY)
@@ -342,8 +350,10 @@ set_property(TARGET lcf PROPERTY WINDOWS_EXPORT_ALL_SYMBOLS ON)
 set_property(TARGET lcf PROPERTY EXPORT_NAME liblcf)
 
 # inih
+if(NOT LIBLCF_FOR_SERVER)
 find_package(inih REQUIRED)
 target_link_libraries(lcf inih::inih)
+endif()
 
 # icu
 set(LCF_SUPPORT_ICU 0)
diff --git a/dummy.cpp b/dummy.cpp
new file mode 100644
index 0000000..e69de29
