diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02e66a1f7..9a08c10e1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -768,24 +768,6 @@ elseif(${PLAYER_TARGET_PLATFORM} STREQUAL "wii")
 		src/platform/sdl/axis.h
 		src/platform/sdl/sdl_ui.cpp
 		src/platform/sdl/sdl_ui.h)
-<<<<<<< HEAD
-elseif(${PLAYER_TARGET_PLATFORM} STREQUAL "amigaos4")
-	if(NOT AMIGAOS4)
-		message(FATAL_ERROR "Missing toolchain file! Use '-DCMAKE_TOOLCHAIN_FILE=path/to/ppc-amigaos.cmake' option.")
-	endif()
-	find_package(SDL REQUIRED)
-	target_compile_definitions(${PROJECT_NAME} PUBLIC USE_SDL=1)
-	target_include_directories(${PROJECT_NAME} PUBLIC ${SDL_INCLUDE_DIR})
-	target_sources(${PROJECT_NAME} PRIVATE
-		src/platform/sdl/sdl_audio.cpp
-		src/platform/sdl/sdl_audio.h
-		src/platform/sdl/axis.h
-		src/platform/sdl/sdl_ui.cpp
-		src/platform/sdl/sdl_ui.h)
-=======
-else()
-	message(FATAL_ERROR "Invalid target platform")
->>>>>>> faa950266
 endif()
 
 # Shared by homebrew platforms
diff --git a/builds/android/app/src/main/java/org/easyrpg/player/button_mapping/InputLayout.java b/builds/android/app/src/main/java/org/easyrpg/player/button_mapping/InputLayout.java
index c5a10cc42..200107032 100644
--- a/builds/android/app/src/main/java/org/easyrpg/player/button_mapping/InputLayout.java
+++ b/builds/android/app/src/main/java/org/easyrpg/player/button_mapping/InputLayout.java
@@ -79,11 +79,8 @@ public class InputLayout {
     private static LinkedList<VirtualButton> getDefaultHorizontalButtonList(Activity activity) {
         LinkedList<VirtualButton> l = new LinkedList<>();
         l.add(new MenuButton(activity, 0.01, 0.01, 90));
-<<<<<<< HEAD
-        l.add(VirtualButton.Create(activity, VirtualButton.KEY_CHAT, 0.90, 0.01, 80));
-=======
->>>>>>> faa950266
         l.add(new VirtualCross(activity, 0.01, 0.4, 100));
+        l.add(VirtualButton.Create(activity, VirtualButton.KEY_CHAT, 0.90, 0.01, 80));
         l.add(VirtualButton.Create(activity, VirtualButton.ENTER, 0.80, 0.55, 100));
         l.add(VirtualButton.Create(activity, VirtualButton.CANCEL, 0.90, 0.45, 100));
 
@@ -93,11 +90,8 @@ public class InputLayout {
     private static LinkedList<VirtualButton> getDefaultVerticalButtonList(Activity activity) {
         LinkedList<VirtualButton> l = new LinkedList<>();
         l.add(new MenuButton(activity, 0.01, 0.5, 90));
-<<<<<<< HEAD
-        l.add(VirtualButton.Create(activity, VirtualButton.KEY_CHAT, 0.70, 0.5, 80));
-=======
->>>>>>> faa950266
         l.add(new VirtualCross(activity, 0.05, 0.65, 100));
+        l.add(VirtualButton.Create(activity, VirtualButton.KEY_CHAT, 0.70, 0.5, 80));
         l.add(VirtualButton.Create(activity, VirtualButton.ENTER, 0.60, 0.75, 100));
         l.add(VirtualButton.Create(activity, VirtualButton.CANCEL, 0.70, 0.65, 100));
 
diff --git a/src/game_map.cpp b/src/game_map.cpp
index 28ee3e741..c441da98a 100644
--- a/src/game_map.cpp
+++ b/src/game_map.cpp
@@ -145,11 +145,8 @@ void Game_Map::Quit() {
 	Dispose();
 	common_events.clear();
 	interpreter.reset();
-<<<<<<< HEAD
-	GMI().MapQuit();
-=======
 	map_cache.reset();
->>>>>>> faa950266
+	GMI().MapQuit();
 }
 
 int Game_Map::GetMapSaveCount() {
diff --git a/src/player.cpp b/src/player.cpp
index e3679b66a..4091e0ff4 100644
--- a/src/player.cpp
+++ b/src/player.cpp
@@ -17,6 +17,8 @@
 
 // Headers
 
+#include <csignal>
+
 #include <algorithm>
 #include <cstring>
 #include <cstdlib>
@@ -24,11 +26,6 @@
 #include <iomanip>
 #include <fstream>
 #include <memory>
-<<<<<<< HEAD
-#include <thread>
-#include <csignal>
-=======
->>>>>>> faa950266
 
 #ifdef _WIN32
 #  include "platform/windows/utils.h"
